java/sage/media/format/FormatParser.java

```diff
@@ -63,7 +63,7 @@
   private static boolean DISABLE_FORMAT_DETECTION = false;
   private static boolean MINIMIZE_EXIF_MEM_USAGE = false;
   public static final java.io.File FORMAT_DETECT_MOUNT_DIR = new java.io.File("/tmp/formatmount");
-  private static String substituteName(String s)
+  public static String substituteName(String s)
```

Comment:
I made it public so that it could be utilized in the Format plugin.  I used it myself to change names into friendly names that Sage wants.

Would you prefer that I did not do that?  Or should I expose it in a different way?

Reply:
OK, if that's why you did it, then it's OK.